yb-voyager/src/query/queryparser/helpers_struct.go

```diff
@@ -59,13 +60,12 @@ func IsCollationObject(parseTree *pg_query.ParseResult) bool {
 	return ok && collation.Kind == pg_query.ObjectType_OBJECT_COLLATION
 }
 
-func GetIndexObjectNameFromIndexStmt(stmt *pg_query.IndexStmt) string {
+func GetIndexObjectNameFromIndexStmt(stmt *pg_query.IndexStmt, dbType string) *sqlname.ObjectNameQualifiedWithTableName {
 	indexName := stmt.Idxname
 	schemaName := stmt.Relation.GetSchemaname()
 	tableName := stmt.Relation.GetRelname()
-	fullyQualifiedName := utils.BuildObjectName(schemaName, tableName)
-	displayObjName := fmt.Sprintf("%s ON %s", indexName, fullyQualifiedName)
-	return displayObjName
+	sqlName := sqlname.NewObjectNameQualifiedWithTableName(dbType, "", indexName, schemaName, tableName)
```

Comment:
should we pass the default schema as public here? considering this would always be postgres?